kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/Comments.java

```diff
@@ -126,4 +126,12 @@
   //- OverrideDoc.node/kind doc
   //- OverrideDoc param.0 _Override
   //- OverrideDoc.text "This documents {@link #compareTo()} over {@link [Override]}. "
+
+  //- @+3fooFunction defines/binding FooFunction
+
+  // This documents FooFunction
```

Comment:
Is there a similar test for a class?

Reply:
yea there's already a test for that up on line 71 I think.